client/js/lounge.js

```diff
@@ -1039,9 +1042,7 @@ $(function() {
 			socket.emit("names", {target: self.data("id")});
 		}
 
-		if (screen.width > 768 && chan.hasClass("chan")) {
-			input.focus();
-		}
+		focus();
```

Comment:
Actually, this never works on my phone (Nexus 4, stock Android, stock Chrome). After I send a message, focus is set to the input. I'm guessing the detection failed there.